pkg/node/server.go

```diff
@@ -36,19 +36,19 @@ func (server *Server) Query(ctx context.Context, req *pb.QueryRequest) (*pb.Quer
 	if req.GetPingback() {
 		_, err = server.dht.Ping(ctx, *req.Sender)
 		if err != nil {
+			server.log.Debug("connection to node failed", zap.Error(err), zap.String("nodeID", req.Sender.Id.String()))
 			err = rt.ConnectionFailed(req.Sender)
 			if err != nil {
 				server.log.Error("could not respond to connection failed", zap.Error(err))
 			}
-			server.log.Debug("connection to node failed", zap.Error(err), zap.String("nodeID", req.Sender.Id.String()))
-		}
-
-		err = rt.ConnectionSuccess(req.Sender)
-		if err != nil {
-			server.log.Error("could not respond to connection success", zap.Error(err))
+		} else {
+			err = rt.ConnectionSuccess(req.Sender)
+			if err != nil {
+				server.log.Error("could not respond to connection success", zap.Error(err))
+			} else {
+				server.log.Sugar().Infof("Successfully connected with %s", req.Sender.Address.Address)
```

Comment:
do we want to log this all the time?

Reply:
at least for now it's nice. maybe we move it to debug later

Reply:
maybe debug?